src/components/AppNavigation/AppNavigation.vue

```diff
@@ -1,77 +1,31 @@
-    <cv-side-nav-items>
-      <cv-side-nav-link to="/">
-        <template slot="nav-icon"><Analytics16 /></template>
-        Overview
-      </cv-side-nav-link>
-      <cv-side-nav-menu title="Health">
-        <template slot="nav-icon"><DataCheck16 /></template>
-        <cv-side-nav-menu-item href="javascript:void(0)">
-          Event log
-        </cv-side-nav-menu-item>
-        <cv-side-nav-menu-item href="javascript:void(0)">
-          Hardware status
-        </cv-side-nav-menu-item>
-        <cv-side-nav-menu-item href="javascript:void(0)">
-          Sensors
-        </cv-side-nav-menu-item>
-      </cv-side-nav-menu>
-      <cv-side-nav-menu title="Control">
-        <template slot="nav-icon"><SettingsAdjust16 /></template>
-        <cv-side-nav-menu-item href="javascript:void(0)">
-          Server power operations
-        </cv-side-nav-menu-item>
-        <cv-side-nav-menu-item href="javascript:void(0)">
-          Manage power usage
-        </cv-side-nav-menu-item>
-        <cv-side-nav-menu-item href="javascript:void(0)">
-          Server LED
-        </cv-side-nav-menu-item>
-      </cv-side-nav-menu>
-      <cv-side-nav-menu title="Configuration">
-        <template slot="nav-icon"><Settings16 /></template>
-        <cv-side-nav-menu-item href="javascript:void(0)">
-          Network setetings
-        </cv-side-nav-menu-item>
-        <cv-side-nav-menu-item href="javascript:void(0)">
-          SNMP settings
-        </cv-side-nav-menu-item>
-        <cv-side-nav-menu-item href="javascript:void(0)">
-          Firmware
-        </cv-side-nav-menu-item>
-      </cv-side-nav-menu>
-      <cv-side-nav-menu title="Access Control">
-        <template slot="nav-icon"><Password16 /></template>
-        <cv-side-nav-menu-item href="javascript:void(0)">
-          LDAP
-        </cv-side-nav-menu-item>
-        <cv-side-nav-menu-item to="/access-control/local-user-management">
-          Local user management
-        </cv-side-nav-menu-item>
-        <cv-side-nav-menu-item href="javascript:void(0)">
-          SSL Certificates
-        </cv-side-nav-menu-item>
-      </cv-side-nav-menu>
-    </cv-side-nav-items>
-  </cv-side-nav>
+  <nav>
+    <ul>
+      <li><router-link to="/">Overview</router-link></li>
+      <li>
+        <router-link to="/access-control/local-user-management"
+          >Local User Management</router-link
+        >
+      </li>
+    </ul>
+  </nav>
 </template>
 
 <script>
-import Analytics16 from "@carbon/icons-vue/es/analytics/16";
-import DataCheck16 from "@carbon/icons-vue/es/data--check/16";
-import SettingsAdjust16 from "@carbon/icons-vue/es/settings--adjust/16";
-import Settings16 from "@carbon/icons-vue/es/settings/16";
-import Password16 from "@carbon/icons-vue/es/password/16";
+// import Analytics16 from "@carbon/icons-vue/es/analytics/16";
+// import DataCheck16 from "@carbon/icons-vue/es/data--check/16";
+// import SettingsAdjust16 from "@carbon/icons-vue/es/settings--adjust/16";
```

Comment:
Why comment out? Why not just remove?

Reply:
I will be pulling them back in for the navigation